src/components/draggable/draggable-item.style.js

```diff
@@ -1,14 +1,28 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import { padding, margin } from "styled-system";
-
 import { baseTheme } from "../../style/themes";
 import Icon from "../icon";
 
 const StyledDraggableContainer = styled.div`
+  ${({ as }) =>
+    as === "ul" &&
+    css`
+      padding-inline-start: 0;
+      padding-right: 4px;
+    `};
   ${margin}
 `;
 
 const StyledDraggableItem = styled.div`
+  ${({ as }) =>
+    as === "li" &&
+    css`
+      border-bottom: none !important;
+      padding: 0 !important;
```

Comment:
Do you need this as seems to be overriding this override?
https://github.com/Sage/carbon/blob/e7bcb2c352768bd58127dbc9ec440c13452584d5/src/components/duelling-picklist/duelling-picklist.style.js#L66